adsp.freesurround/resources/language/English/strings.po

```diff
@@ -22,15 +22,15 @@ msgid "Free Surround"
 msgstr ""
 
 msgctxt "#30001"
-msgid "Free Surround ..."
+msgid "Free Surround..."
 msgstr ""
 
 msgctxt "#30002"
-msgid "FreeSurround, a new surround processor. Is based on foo_dsp_fsurround from pro_optimizer on foobar2000 0.9.x."
+msgid "FreeSurround, is a new surround processor. It's based on foo_dsp_fsurround found on pro_optimizer in foobar2000 0.9.x."
```

Comment:
Second sentence is still wrong somehow.
"It's based on foo_dsp_fsurround as found in pro_optimizer from foobar2000 0.9.x."  or something like that